### ISSUE TITLE
robotparser = "0.11" is not released

### ISSUE THREAD

**Issue by j-mendez** [messense/robotparser-rs#32]
Thank you for this repo and providing a clean interface to manage parsing the robots.txt file. It seems the latest version with the changes that handle lower casing the fields to properly get the values is not yet published. I am going to publish a fork until then called `robotparser-fork`. Thanks again for the project!

**Comment by messense** [comment]
I'll do a release soon.

**Comment by messense** [comment]
https://crates.io/crates/robotparser/0.11.0

**Comment by j-mendez** [comment]
@messense thanks for the quickness on this! v0.11 has a bit of changes from the v0.10 upgrade. Look forward to converting the usage in the https://github.com/madeindjs/spider lib since we use it to handle the robots file. I made some changes in my fork that works off of v0.10.2. I thought it would be nice to have the ability to set a UA to avoid being blocked by certain sites. Heres a commit of the minor fork changes https://github.com/j-mendez/robotparser-rs/commit/8759dec8ce68cd5dd2bd5b178e576cf00a8f366c.

**Comment by j-mendez** [comment]
@messense there was a small issue with `*` not being captured on 0.10.2. Not sure if it was fixed in v0.11 but, the fix is on this line https://github.com/j-mendez/robotparser-rs/commit/8759dec8ce68cd5dd2bd5b178e576cf00a8f366c#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R468.

**Comment by messense** [comment]
Feel free to open a PR for that change.
